addons/config/expected/cni.tanzu.vmware.com/v1alpha1/antreaconfig/default.yaml

```diff
@@ -8,18 +8,57 @@ metadata:
 spec:
   antrea:
     config:
-      trafficEncapMode: encap
-      noSNAT: false
-      tlsCipherSuites: TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_RSA_WITH_AES_256_GCM_SHA384
```

Comment:
Why is this being removed?

Reply:
the template here is an old version - this is the default value of the field set by the antrea controller.